equinix/resource_fabric_service_profile_acc_test.go

```diff
@@ -17,139 +17,134 @@ import (
 	v4 "github.com/equinix-labs/fabric-go/fabric/v4"
 )
 
-func testAccFabricReadServiceProfileConfig(uuid string) string {
-	return fmt.Sprintf(`data "equinix_fabric_service_profile" "test" {
-	uuid = "%s"
-	}
-`, uuid)
-}
-
-func testAccFabricReadServiceProfilesListConfig(name string) string {
-	return fmt.Sprintf(`data "equinix_fabric_service_profiles" "test" {
-		filter {
-			property = "/name"
-			operator = "="
-			values = ["%s"]
-		}
-	}
-`, name)
-}
-
-func TestAccFabricReadServiceProfileByUuid(t *testing.T) {
+func TestAccFabricCreateServiceProfile_SP_FCR_ba(t *testing.T) {
+	spName1 := "fabric_tf_acc_test_01"
+	spName2 := "fabric_tf_acc_test_02"
+	typel2 := "L2_PROFILE"
```

Comment:
These constants can be placed directly in the config string, they don't need to be setup here.

Reply:
done